[New Product] : Gleam (#8371)
captn3m0/endoflife.date: products/gleam.md

---
title: Gleam
addedAt: 2025-09-09
category: lang

> iconslug: gleam

Reviewer: `s` should be capital.

```suggestion
iconSlug: gleam
```

permalink: /gleam
changelogTemplate: https://github.com/gleam-lang/gleam/releases/tag/v__LATEST__
eolColumn: Support

> 

Reviewer: You'll need to add identifiers.

https://repology.org/project/gleam/versions seems to have most covered, but you should add `github` for source code, and https://github.com/gleam-lang/gleam/pkgs/container/gleam as well. The contributing page, and https://endoflife.date/help/identifiers-needed/ for more guidance.

> 

Reviewer: Please add a `versionCommand`.

    eol: 2020-08-28
    latest: "0.10.1"
    latestReleaseDate: 2020-07-15

  - releaseCycle: "0.9"
    releaseDate: 2020-06-01
    eol: 2020-07-01
    latest: "0.9.1"
    latestReleaseDate: 2020-06-12

  - releaseCycle: "0.8"
    releaseDate: 2020-05-07
    eol: 2020-06-01
    latest: "0.8.1"
    latestReleaseDate: 2020-05-19

  - releaseCycle: "0.7"
    releaseDate: 2020-03-01
    eol: 2020-05-07
    latest: "0.7.1"
    latestReleaseDate: 2020-03-08

  - releaseCycle: "0.6"
    releaseDate: 2019-12-25
    eol: 2020-03-01
    latest: "0.6.0"
    latestReleaseDate: 2019-12-25

  - releaseCycle: "0.5"
    releaseDate: 2019-12-17
    eol: 2019-12-25
    latest: "0.5.1"
    latestReleaseDate: 2019-12-23

  - releaseCycle: "0.4"
    releaseDate: 2019-09-19
    eol: 2019-12-17
    latest: "0.4.2"
    latestReleaseDate: 2019-10-23

  - releaseCycle: "0.3"
    releaseDate: 2019-08-09
    eol: 2019-09-19
    latest: "0.3.0"
    latestReleaseDate: 2019-08-09

  - releaseCycle: "0.2"
    releaseDate: 2019-06-25
    eol: 2019-08-09
    latest: "0.2.0"
    latestReleaseDate: 2019-06-25

  - releaseCycle: "0.1"
    releaseDate: 2019-04-15
    eol: 2019-06-25
    latest: "0.1.2"

>     latestReleaseDate: 2019-05-12

Reviewer: We will need to drop these, as [Gleam v1](https://gleam.run/news/gleam-version-1/) was the first "stable" release, and we do not list non-stable releases.

```suggestion
```

---

> > [Gleam](https://gleam.run/) is a friendly language for building type-safe, scalable systems. It runs on the battle-tested Erlang virtual machine—a platform that powers planet-scale services—which ensures it's ready for workloads of any size.

Reviewer: ```suggestion
> [Gleam](https://gleam.run/) is a programming language for building type-safe, scalable systems. It runs on the [Erlang](/erlang) VM and is part of the BEAM ecosystem.
```

We try to avoid fluffy adjectives, and stick to the basics. The assumption is that the reader already knows what gleam is, and probably uses it as well.
